cloudflare/resource_cloudflare_logpush_job.go

```diff
@@ -0,0 +1,176 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceCloudflareLogpushJob() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareLogpushJobCreate,
+		Read:   resourceCloudflareLogpushJobRead,
+		Update: resourceCloudflareLogpushJobUpdate,
+		Delete: resourceCloudflareLogpushJobDelete,
+
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"zone": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"logpull_options": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"destination_conf": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"ownership_challenge": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+		},
+	}
+}
+
+func getJobFromResource(d *schema.ResourceData) cloudflare.LogpushJob {
+	id, err := strconv.Atoi(d.Id())
+
+	if err != nil {
+		fmt.Errorf("Could not extract Logpush job from resource - invalid identifier: %+v", id)
```

Comment:
If you want these to be visible in the Terraform logs, you'll need to use `log.Printf("[INFO] ... ")` otherwise it gets swallowed up.

Reply:
(goes for all `fmt.Errorf` calls in this PR)

Reply:
Thanks - tuned a couple in a793a42. Thinking of leaving the others as is for consistency with other resources.